lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -676,6 +726,147 @@
             ]
          }
       },
+      {
+         "type": "Microsoft.Insights/autoscaleSettings",
+         "apiVersion": "2015-04-01",
```

Comment:
update apiVersion

Reply:
2015-04-01 is the latest apiVersion. When I execute this Powershell
`((Get-AzureRmResourceProvider -ProviderNamespace Microsoft.Insights).ResourceTypes | Where-Object ResourceTypeName -eq autoscalesettings).apiversions`
I get this list
```
2015-04-01
2014-04-01
```